src/test/java/com/example/demo/repository/BookShelfRepositoryTest.java

```diff
@@ -0,0 +1,21 @@
+package com.example.demo.repository;
+
+import com.example.demo.model.Book;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class BookShelfRepositoryTest {
+
+    @Test
+    void addBook() {
```

Comment:
This test doesn't check the work of the method addBook

Reply:
it is, through debug we checked that both books had been added

Reply:
Ok)